### ISSUE TITLE
Clickhouse 'OpenTelemetry Support' can not write to jaeger-collector

### ISSUE THREAD

**Issue by godliness** [ClickHouse/ClickHouse#30838]
clickhouse-server(v21.9.5.16-stable) shows error:
```
2021.10.29 08:36:31.416694 [ 59 ] {} <Error> void DB::SystemLog<DB::OpenTelemetrySpanLogElement>::flushImpl(const std::vector<LogElement> &, uint64_t) [LogElement = DB::OpenTelemetrySpanLogElement]: Code: 86. DB::Exception: Received error from remote server /api/v2/spans. HTTP status code: 400 Bad Request, body: Cannot parse Content-Type: mime: no media type
: while writing suffix to view default.zipkin_spans. (RECEIVED_ERROR_FROM_REMOTE_IO_SERVER), Stack trace (when copying this message, always include the lines below):

0. DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int, bool) @ 0x93a6a9a in /usr/bin/clickhouse
1. DB::assertResponseIsOk(Poco::Net::HTTPRequest const&, Poco::Net::HTTPResponse&, std::__1::basic_istream<char, std::__1::char_traits<char> >&, bool) @ 0xea933e3 in /usr/bin/clickhouse
2. DB::PushingToSinkBlockOutputStream::writeSuffix() @ 0x10b3207f in /usr/bin/clickhouse
3. DB::PushingToViewsBlockOutputStream::writeSuffix() @ 0x10b3b343 in /usr/bin/clickhouse
4. ? @ 0x10b3ea2f in /usr/bin/clickhouse
5. DB::runViewStage(DB::ViewRuntimeData&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::function<void ()>) @ 0x10b37122 in /usr/bin/clickhouse
6. DB::PushingToViewsBlockOutputStream::writeSuffix() @ 0x10b3b64d in /usr/bin/clickhouse
7. DB::SystemLog<DB::OpenTelemetrySpanLogElement>::flushImpl(std::__1::vector<DB::OpenTelemetrySpanLogElement, std::__1::allocator<DB::OpenTelemetrySpanLogElement> > const&, unsigned long) @ 0x10e60117 in /usr/bin/clickhouse
8. DB::SystemLog<DB::OpenTelemetrySpanLogElement>::savingThreadFunction() @ 0x10e5e060 in /usr/bin/clickhouse
9. ThreadFromGlobalPool::ThreadFromGlobalPool<DB::SystemLog<DB::OpenTelemetrySpanLogElement>::startup()::'lambda'()>(DB::OpenTelemetrySpanLogElement&&, DB::SystemLog<DB::OpenTelemetrySpanLogElement>::startup()::'lambda'()&&...)::'lambda'()::operator()() @ 0x10e5df16 in /usr/bin/clickhouse
10. ThreadPoolImpl<std::__1::thread>::worker(std::__1::__list_iterator<std::__1::thread, void*>) @ 0x93e7abf in /usr/bin/clickhouse
11. ? @ 0x93eb3a3 in /usr/bin/clickhouse
12. start_thread @ 0x9609 in /usr/lib/x86_64-linux-gnu/libpthread-2.31.so
13. clone @ 0x122293 in /usr/lib/x86_64-linux-gnu/libc-2.31.so
 (version 21.9.5.16 (official build))
```
Depends on link: https://clickhouse.com/docs/zh/operations/opentelemetry/#integration-with-monitoring-systems
```
CREATE MATERIALIZED VIEW default.zipkin_spans
ENGINE = URL('http://simplest-collector.user01.svc.cluster.local:9411/api/v2/spans', 'JSONEachRow')
SETTINGS output_format_json_named_tuples_as_objects = 1,
    output_format_json_array_of_rows = 1 AS
SELECT
    lower(hex(reinterpretAsFixedString(trace_id))) AS traceId,
    lower(hex(parent_span_id)) AS parentId,
    lower(hex(span_id)) AS id,
    operation_name AS name,
    start_time_us AS timestamp,
    finish_time_us - start_time_us AS duration,
    cast(tuple('clickhouse'), 'Tuple(serviceName text)') AS localEndpoint,
    cast(tuple(
        attribute.values[indexOf(attribute.names, 'db.statement')]),
        'Tuple("db.statement" text)') AS tags
FROM system.opentelemetry_span_log
```
Set up jaeger as below:
```
jaeger-collector with options:
      collector.zipkin.host-port: ":9411"
      collector.zipkin.allowed-origins: "*"
```

Seems like clickhouse write to jaeger without header "content-type"?

**Comment by qoega** [comment]
Known issue #27027

**Comment by FrankChen021** [comment]
This problem has been addressed by #32113 and #32112 together.
